src/definitions/internet.ts

```diff
@@ -29,4 +29,6 @@ export type InternetDefinition = LocaleEntry<{
    * List of some HTTP status codes.
    */
   http_status_code: Record<HTTPStatusCodeType, number[]>;
+
+  jwt: Record<'alg', string[]>;
```

Comment:
Please add a minimalistic jsdocs description.

And maybe use an object instead of a record:

```suggestion
  jwt: {
    /** 
     * description
     */
    algorithm: string[]
  }
```

Not sure about whether we have to nest the jwt object here at all. I have to think about that some more.

Reply:
I agree. I just removed the nesting, for now.